src/nvim/api/window.c

```diff
@@ -19,6 +19,45 @@
 #include "nvim/window.h"
 #include "nvim/screen.h"
 #include "nvim/move.h"
+#include "nvim/fold.h"
+
+/// Create a fold from line "start" to line "end" (inclusive) in the current
+/// window.
+/// @param window
```

Comment:
standard docstring for this is:
```suggestion
/// @param window   Window handle, or 0 for current window
```